create.go

```diff
@@ -72,6 +72,10 @@ func create(containerID, bundlePath, console, pidFilePath string) error {
 		return err
 	}
 
+	if pidFilePath == "" {
```

Comment:
I appreciate that checks are performed later on, but the point is that *work has been done **before** those checks are carried out*. Work that we only find out should **not** have been done much later on in the program. Are you suggesting that this sort of approach is "the golang way"?